apps/mobile/src/styles/global-styles/margins.ts

```diff
@@ -0,0 +1,54 @@
+import { SPACINGS } from "~/styles/global-styles/spacings";
+
+const MARGINS = {
+	mXs: { margin: SPACINGS.xs },
```

Comment:
For the spacing names T-short is good, but for the padding/margin values let's use more readable approach:
`const MARGINS = {
	m0: { margin: 0},
  m1: { margin: SPACING.xxxs }, 
....
`